views/index.ejs

```diff
@@ -168,6 +220,76 @@
       </section>
       <%- include('partials/testimonial') %>
     </section>
+    <!-- Feedback Modal -->
+<!-- Feedback Modal -->
+<div class="modal fade" id="feedbackModal" tabindex="-1" role="dialog" aria-labelledby="feedbackModalLabel" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="feedbackModalLabel">We'd Love Your Feedback!</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <form id="feedbackForm">
+          <div class="form-group">
+            <label for="feedbackText">Your Feedback</label>
+            <textarea class="form-control" id="feedbackText" rows="4" placeholder="Let us know what you think..." required></textarea>
+          </div>
+          <div class="form-group">
+            <label for="email" style="margin-bottom: 5px;">Your Email (optional)</label>
+            <input type="email" style="margin-bottom: 10px;" class="form-control" id="email" placeholder="Enter your email">
+          </div>
+          <button type="submit" class="btn btn-success btn-block">Submit Feedback</button>
+        </form>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+
+<script>
+ $(document).ready(function() {
+    // Show the feedback modal after 15 seconds
+    setTimeout(function() {
```

Comment:
This will annoy our users as asking for feedback after 15 sec every time the page is loaded is not good for UX